src/components/AddBookingForm/AddBookingForm.jsx

```diff
@@ -23,21 +38,46 @@ const AddBookingForm = ({ bookings, setBookings }) => {
     setFormData((prevData) => ({ ...prevData, [name]: value }));
   };
 
+  const validateBooking = () => {
+    if (!isNaN(parseInt(formData.firstName))) {
+      alert("Enter a valid name");
+    } else if (!isNaN(parseInt(formData.surname))) {
+      alert("Enter a valid surname");
+    } else if (
+      !(
+        formData.email.includes("@") &&
+        formData.email.includes(".") &&
+        formData.email.indexOf("@") === formData.email.lastIndexOf("@") &&
+        formData.email.indexOf("@") < formData.email.lastIndexOf(".")
+      )
```

Comment:
This works but it's fairly hard to read I normally use a regex in these cases but that is a diffrent thing. A fairly simple regex such as formData.email.match(/^.+@.+\.$/)) should work